low-level/files/opt/seagate/sspl/setup/resource_map/resource_map.py

```diff
@@ -67,3 +68,24 @@ def get_health_template(uid, is_fru: bool):
                 "specifics": []
             }
         }
+
+    @staticmethod
+    def set_health_data(health_data: dict, status, description=None,
+                        recommendation=None, specifics=None):
+        """Sets health attributes for a component."""
+        good_state = (status == "OK")
+        if not description:
+            description = "%s %s in good health" % (
+                health_data.get("uid"),
+                'is' if good_state else 'is not')
+        if not recommendation:
+            recommendation = 'None' if good_state\
+                             else "Fault detected, please contact Seagate support."
```

Comment:
pickup from default recommendation constant, where currently it says to contact seagate support, tomorrow we may want to change that too to refer LR User Guide / Troubleshooting guide. So its imp we pick up default strings from common places

Reply:
Yes, already done in other PR's, just included to make code working.

Reply:
What do you mean other PR? Other PR having same function referring from default recommendation constant ?
Then track this and make sure at last we have that in place, at no cost this code should over write changes in other PR, keep an eye

Reply:
addressed here only. 👍